godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -227,6 +231,23 @@ class TractViewController: BaseViewController {
     
     // MARK: - Handle page movements
     
+    func moveToPage(notification: Notification) {
+        guard let dictionary = notification.object as? [String: String] else {
```

Comment:
Custom Information should be passed in the `userInfo` property, not in the `object`.